streams/shared/src/main/scala/zio/stream/experimental/ZChannel.scala

```diff
@@ -456,6 +456,12 @@ sealed trait ZChannel[-Env, -InErr, -InElem, -InDone, +OutErr, +OutElem, +OutDon
   final def mapError[OutErr2](f: OutErr => OutErr2): ZChannel[Env, InErr, InElem, InDone, OutErr2, OutElem, OutDone] =
     catchAllCause((cause: Cause[OutErr]) => cause.failureOrCause.fold(err => ZChannel.fail(f(err)), ZChannel.halt(_)))
```

Comment:
Let's delegate to mapErrorCause while you're here :-)

Reply:
Done